crc/services/workflow_service.py

```diff
@@ -734,6 +734,12 @@ def spiff_task_to_api_task(spiff_task, add_docs_and_forms=False):
                 # Otherwise, just use the curreent title.
         elif task.title and ' ' in task.title:
             task.title = task.title.partition(' ')[2]
+
+        if task.properties and "clear_data" in task.properties:
```

Comment:
@Leashys - this won't work for Alex's case that he described this morning.  
What we need to do is /prevent/ the previously submitted form data from overwriting what he has set in a previous step.

Reply:
a good test for this is
Task 1:  Have a radio button for "a", "b", "c"
Task 2: Display a text box that contains the value selected on the previous page.
Task 3: End Task

It should be possible to change the value in Task 1, and have that change reflected in task 2.